eZ/Bundle/EzPublishCoreBundle/EventListener/ViewControllerListener.php

```diff
@@ -59,7 +59,7 @@ public function __construct(
 
     public static function getSubscribedEvents()
     {
-        return array( KernelEvents::CONTROLLER => 'getController' );
+        return array( KernelEvents::CONTROLLER => array( 'getController', 10 ) );
```

Comment:
it was using short array syntax in @lolautruche 's original fix. But we decided to use it like that in case we want to backport this feature so I changed it.